doc/source/data/advanced-pipelines.rst

```diff
@@ -140,6 +140,14 @@ Transformations made prior to the Dataset prior to the call to ``.repeat()`` are
 
 For example, in the following pipeline, the ``map(func)`` transformation only occurs once. However, the random shuffle is applied to each repetition in the pipeline.
 
+.. note::
```

Comment:
Should this be a warning, or is a note strong enough?

We could also more strongly de-emphasize global shuffling in the pipelining examples across the board, but I don't think we should do that until we have a viable local shuffling alternative: https://github.com/ray-project/ray/issues/24159

Reply:
I'd vote for note:
- this is Advanced Pipelines page (unlike Getting Started or other more basic venues)
- this is still a quite unique value we can provide
- with new shuffle algo, the cost issue should be much alleviated (albeit shuffle is anyway more costly -- but it will be easier for users to justify with the known benefits).